conf/messages

```diff
@@ -28,7 +28,8 @@ agent.header.serviceName                            = Manage your client’s Inc
 
 # HTML Titles
 titlePattern.serviceName.govUk                      = Manage your Income Tax updates
```

Comment:
This can be removed and any references updated to either htmlTitle or htmlTitle.agent